cmd/git-sync/main.go

```diff
@@ -371,9 +384,11 @@ func main() {
 			continue
 		} else if changed && webhook != nil {
 			webhook.Send(hash)
```

Comment:
What's the reasoning behind no success increment within the webhook case?

Reply:
A bug?  Will fix